web-app/src/app/(landing)/(auth)/signup/data.ts

```diff
@@ -2,30 +2,35 @@ import { z } from "zod";
 
 import { FormData, FormIntlTranslation } from "@/lib/form";
 
-const signUpFormSchema = (t: FormIntlTranslation<"Auth.Pages.SignUp.Form">) =>
+const signUpFormSchema = (
+  t: FormIntlTranslation<"Auth.Pages.SignUp.Form"> | undefined = undefined,
+) =>
   z
     .object({
       username: z
         .string()
         .min(2, {
-          message: t("Errors.Username.min"),
+          message: t?.("Errors.Username.min"),
```

Comment:
Same here, do we want a fallback message or require the formintl

Reply:
See https://github.com/masumi-network/sokosumi/pull/112#discussion_r2007241436